src/main/webapp/timestamps.js

```diff
@@ -42,11 +41,8 @@ function timestampToSeconds(timestampMs) {
 }
 
 /**
- * @return {number} The {@code date} in seconds.
+ * Converts `seconds` into milliseconds.
  */
-// TODO: Remove this method once the pull request updating
-// Date to long is approved.
-function getDateInSeconds(date) {
-  return date.getSeconds() + date.getMinutes() * MINUTES_TO_SECONDS +
-      date.getHours() * HOURS_TO_SECONDS;
+function secondsToMilliseconds(seconds) {
```

Comment:
wasn't this code added by Nathan? Should this PR be synced?

Reply:
No, this was a function I added. His function converts milliseconds to seconds (the other way around)